packages/core/src/navigator/drawer/DrawerContent.js

```diff
@@ -192,6 +222,44 @@ const DrawerContent = ({
     );
   }
 
+  if (
+    minimalRequiredMobileAppVersion?.activate === true &&
+    mobileVersion < minimalRequiredVersion
+  ) {
+    return (
+      <PopUp
+        visible={true}
+        title={I18n.t('Base_Information')}
+        data={I18n.t('Base_MinimalRequiredVersion')}>
+        <View style={styles.btnContainer}>
+          <IconButton
+            title={I18n.t('Base_Update')}
+            iconName="angle-double-up"
+            color={Colors.primaryColor}
+            onPress={() => handleUpdate()}
+            style={styles.btn}
+          />
+
+          <IconButton
+            title={I18n.t('Base_Refresh')}
+            iconName="refresh"
+            FontAwesome5={false}
+            color={Colors.secondaryColor}
+            onPress={onRefresh}
+            style={styles.btn}
+          />
+          <IconButton
+            title={I18n.t('Base_Logout')}
+            iconName="power-off"
+            color={Colors.errorColor}
+            onPress={() => dispatch(logout())}
+            style={styles.btn}
+          />
+        </View>
+      </PopUp>
```

Comment:
Maybe we can do this in a new component to make it cleaner and remove empty space between components please